components/contract-components/contract-deploy-form/built-in-contract.tsx

```diff
@@ -265,7 +265,7 @@ const BuiltinContractForm: React.FC<BuiltinContractFormProps> = ({
               });
               onSuccess();
               router.push(
-                `/${SupportedChainIdToNetworkMap[selectedChain]}/${UrlMap[contractType]}/${contractAddress}`,
+                `/${SupportedChainIdToNetworkMap[selectedChain]}${UrlMap[contractType]}/${contractAddress}`,
```

Comment:
could we just fix it in this spot? could we just do a `.replaceAll("//", "/")` and be done with it?